src/action/common/place_nix_configuration.rs

```diff
@@ -43,7 +43,7 @@ impl PlaceNixConfiguration {
         );
         let create_directory =
             CreateDirectory::plan(NIX_CONF_FOLDER, None, None, 0o0755, force).await?;
-        let create_file = CreateFile::plan(NIX_CONF, None, None, 0o0664, buf, force).await?;
+        let create_file = CreateFile::plan(NIX_CONF, None, None, 0o100644, buf, force).await?;
```

Comment:
I don't think this is necessary after the latest changes to main (we only check rwx now).